src/docs/spec-formats-tsv.md

```diff
@@ -134,6 +134,28 @@ For any given propagatable slot, condensation is only allowed if (1) all mapping
 Implementations that support condensation MUST also support propagation.
 
 
+## Non-standard slots
+
+If an implementation does not support [non-standard slots](spec-model.md#non-standard-slots), then:
+
+* a SSSOM/TSV reader MUST discard any unknown top-level YAML key in the metadata block, and any unknown TSV column in the TSV section;
+* a SSSOM/TSV reader MUST NOT write any unknown top-level YAML key in the metadata block, or any unknown TSV column in the TSV section.
```

Comment:
- "reader" ... MUST NOT .... "write"
- Also, seems redundant with the above?

Reply:
The second “reader” is indeed a mistake, I’ll fix it.

As for the redundancy, I don’t think so. The spec don’t say anywhere else what implementations are supposed to do with unknown slots.

Reply:
(No need to belabour the point, I thought "if the reader MUST discard, then the "writer" cant really have anything to write anymore - but I guess it does not hurt to be clear)

Reply:
> if the reader MUST discard, then the "writer" cant really have anything to write anymore

A writer can have to write a set that is _not_ the result of a prior parsing operation (e.g. if the set has been built from scratch by the application).

Reply:
Of course!